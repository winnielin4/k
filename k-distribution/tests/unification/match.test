symbolicExp     matches 3               ?
3               matches symbolicExp     ?
3 + symbolicExp matches symbolicExp     ?
symbolicExp     matches 3 + symbolicExp ?
3 + symbolicExp matches 3 + 5           ?
