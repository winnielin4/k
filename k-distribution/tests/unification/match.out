<generatedTop>
  <k>
    .
  </k>
  <results>
    ListItem ( true )
    ListItem ( false )
    ListItem ( false )
    ListItem ( true )
    ListItem ( true )
  </results>
</generatedTop>
